WalletWasabi/WabiSabi/Crypto/CredentialIssuer.cs

```diff
@@ -56,7 +56,7 @@ public class CredentialIssuer
 			ulong maxAmount)
 		{
 			MaxAmount = maxAmount;
-			RangeProofWidth = (int)Math.Log2(MaxAmount);
+			RangeProofWidth = (int)Math.Ceiling(Math.Log2(MaxAmount));
```

Comment:
if `MaxAmount` is a power of two i think there's still an off by 1, `Math.Log2(2^i) == i` but `i` bits can only represent values between `0` and `2^i-1`